config/sql/migration/migrate.23.sql

```diff
@@ -0,0 +1,10 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+SAVEPOINT MIGRATION;
+
+ALTER TABLE ecu_serials RENAME TO ecus;
+CREATE TABLE secondary_ecus(serial TEXT PRIMARY KEY, sec_type TEXT NOT NULL, public_key_type TEXT NOT NULL, public_key TEXT NOT NULL, FOREIGN KEY(serial) REFERENCES ecus ON DELETE CASCADE ON UPDATE CASCADE);
```

Comment:
IMHO, these two tables should not know anything about each other. libaktualizr should not know details of any secondary implementation. In ideal world the IP secondary should have its own dedicated persistent layer.

Reply:
Well the *real* world is all about compromises.